src/components/Modal/Modal.scss

```diff
@@ -1,4 +1,4 @@
-@import 'index.scss';
+@import 'variables.scss';
```

Comment:
I think you could us a master sheet in which you import all your sass sheets, that way you can save yourself from adding common sheets in every other file.